cpp/src/DataStorm/NodeI.cpp

```diff
@@ -489,21 +484,11 @@ shared_ptr<SessionI>
 NodeI::getSession(const Identity& ident) const
 {
     unique_lock<mutex> lock(_mutex);
-    if (ident.category == "s")
+    assert(ident.category == "s");
```

Comment:
This is bogus because `getSession` is also used by `Node::getSessionConnection(string_view ident) const noexcept` which is part of the public API.